libraries/tools/idl2k/src/main/kotlin/build.kt

```diff
@@ -1,13 +1,26 @@
 package org.jetbrains.idl2k
 
-import org.antlr.v4.runtime.ANTLRFileStream
-import kotlin.collections.HashSet
+import org.antlr.v4.runtime.CharStreams
 import java.io.File
 import java.io.IOException
 import java.net.URL
 import java.util.ArrayList
 import java.util.LinkedHashMap
-
+import kotlin.collections.HashSet
+import kotlin.collections.List
+import kotlin.collections.Map
+import kotlin.collections.MutableList
+import kotlin.collections.asSequence
+import kotlin.collections.contains
+import kotlin.collections.emptyMap
+import kotlin.collections.fold
+import kotlin.collections.forEach
+import kotlin.collections.map
+import kotlin.collections.mapValues
+import kotlin.collections.plus
+import kotlin.collections.reduce
+import kotlin.collections.set
+import kotlin.collections.sortedBy
```

Comment:
`kotlin.collections` package is imported by default. Why these imports are required? 😕